docs/getting-started/install-kura.md

```diff
@@ -26,12 +26,13 @@ Installers of type (2) with the suffix `nn` are **No Networking** profiles that
 A user can deploy Kura on a target system using the installer tailored for the device architecture. The installer file looks like:
 
 ```
-kura-<kura-version>_generic-<arch>_installer.deb/rpm
+kura-<kura-version>_<arch>_installer.deb/rpm
 ```
 
-where `<arch>` is one of the **supported architectures**: *x86_64*, *arm32*, and *arm64*. Kura can work on systems that have available the dependencies listed in the [Kura dependencies](#kura-dependencies) section, and that have **at least one** physical ethernet interface.
+where `<arch>` is one of the **supported architectures**: *x86_64*, *armhf*, and *aarch64*. Kura can work on systems that have available the dependencies listed in the [Kura dependencies](#kura-dependencies) section, and that have **at least one** physical ethernet interface. Java 17 is required.
```

Comment:
I would add something more descriptive like:

```suggestion
where `<arch>` is one of the **supported architectures**: *x86_64*, *armhf*, and *aarch64*. Kura can work on systems that have available the dependencies listed in the [Kura dependencies](#kura-dependencies) section, and that have **at least one** physical ethernet interface. 

From Kura version 6.0.0 the Java 17 runtime is **required** to run the framework correctly.
```

Reply:
Addressed in [1babba4](https://github.com/eclipse-kura/kura/pull/5613/commits/1babba4085e7c85acb08879e5db05b594fb1bb4c)